model/src/gateway/intents.rs

```diff
@@ -94,6 +94,24 @@ bitflags! {
         /// [`INVITE_CREATE`]: super::event::Event::InviteCreate
         /// [`INVITE_DELETE`]: super::event::Event::InviteDelete
         const GUILD_INVITES = 1 << 6;
+        /// Guild scheduled events intent.
+        ///
+        /// Event(s) received:
+        ///
+        /// - [`GUILD_SCHEDULED_EVENT_CREATE`]
+        /// - [`GUILD_SCHEDULED_EVENT_UPDATE`]
+        /// - [`GUILD_SCHEDULED_EVENT_DELETE`]
+        /// - [`GUILD_SCHEDULED_EVENT_USER_ADD`]
+        /// - [`GUILD_SCHEDULED_EVENT_USER_REMOVE`]
+        ///
+        /// [`INVITE_CREATE`]: super::event::Event::InviteCreate
+        /// [`INVITE_DELETE`]: super::event::Event::InviteDelete
```

Comment:
```suggestion
```

Reply:
What's the reason of removing those links?

Reply:
I might be mistaken, but they don't seem to link to anything.

Reply:
True 🤔 but that's also the case for every other one.